winch/codegen/src/masm.rs

```diff
@@ -59,6 +59,25 @@ pub(crate) enum CmpKind {
     GeU,
 }
 
+/// Kinds of binary comparison of floats in WebAssembly. The [`MacroAssembler`]
+/// implementation for each ISA is responsible fro emitting the correct
+/// sequence of instructions when lowering to machine code.
+#[derive(Clone)]
+pub(crate) enum FloatCmpKind {
```

Comment:
I'm fine introducing this new enum, but for consistency, I'd expect that we update `CmpKind` as part of this change.

Reply:
What kind of update are you expecting to see to `CmpKind`? A rename to `IntegerCmpKind` or something else?

Reply:
`IntCmpKind` sounds like a good name to me, but I don't feel strongly, `IntegerCmpKind` works well too (just a bit long, but 🤷🏽‍♂️)